### PR TITLE
makes window frames into structures again

### PR DESCRIPTION
**Pull request by Kylerace**
## About The Pull Request
window frames are now structures. also moves north facing frills to using OVER_FRILL_PLANE instead of WALL_FRAME because that was making the frames render over the frills of the windows

## Why It's Good For The Game
theyre dumb as turfs

## Changelog

:cl:
code: makes window frames into structures again
/:cl:

### CODE REVIEW

File: code/datums/elements/frill.dm [hunk 1]
```diff
@@ -36,7 +36,7 @@ GLOBAL_LIST_EMPTY(frill_objects)
 /datum/element/frill/Detach(turf/target)
 
 	target.cut_overlay(get_frill_object(icon_path, target.smoothing_junction, pixel_y = 32))
-	target.cut_overlay(get_frill_object(icon_path, target.smoothing_junction, plane = WALL_PLANE, pixel_y = 32))
+	target.cut_overlay(get_frill_object(icon_path, target.smoothing_junction, pixel_y = 32))
```
Comment (Kylerace) [inline code/datums/elements/frill.dm]:
```suggestion
	target.cut_overlay(get_frill_object(icon_path, target.smoothing_junction, plane = OVER_FRILL_PLANE, pixel_y = 32))
```

**Comment by Kylerace** [comment]
why are you like this

File: code/game/turfs/closed/wall/window_frames.dm [hunk 2]
```diff
@@ -193,43 +178,35 @@
 	else
 		. += "<span class='notice'>The window frame is empty</span>"
 
-/turf/closed/wall/window_frame/proc/on_painted(is_dark_color)
-	SIGNAL_HANDLER
-
-	if (is_dark_color) //Opaque directional windows restrict vision even in directions they are not placed in, please don't do this
-		set_opacity(255)
-	else
-		set_opacity(initial(opacity))
```
Comment (LemonInTheDark) [inline code/game/turfs/closed/wall/window_frames.dm:202]: This logic's still on the window itself ye?

File: code/game/objects/items/devices/radio/intercom.dm [hunk 3]
```diff
@@ -20,7 +20,6 @@
 	if(!current_area)
 		return
 	RegisterSignal(current_area, COMSIG_AREA_POWER_CHANGE, .proc/AreaPowerCheck)
-	AddElement(/datum/element/wall_mount)
```
Comment (LemonInTheDark) [inline code/game/objects/items/devices/radio/intercom.dm:23]: KYLERRRRR (also move this above the area check pls thxs

**Review by LemonInTheDark — CHANGES_REQUESTED** [review]

File: code/game/objects/items/devices/radio/intercom.dm (hunk 3 above)
Comment (Kylerace) [reply code/game/objects/items/devices/radio/intercom.dm:23]: its a duplicate of a line already doing it above it github just isnt showing it in the unexpanded view

Comment (LemonInTheDark) [reply code/game/objects/items/devices/radio/intercom.dm:23]: I hate old me

File: code/game/turfs/closed/wall/window_frames.dm (hunk 2 above)
Comment (Kylerace) [reply code/game/turfs/closed/wall/window_frames.dm:202]: yes, this is actually a remnant from when i was trying to make windows into just the frame, with the actual glass part being the same object. but i gave up on that when i changed frames to turfs. so this does nothing

**Pull request merged**